lib/ramble/ramble/schema/applications.py

```diff
@@ -86,6 +86,11 @@
     'additionalProperties': False
 }
 
+repeats_def = {
+    'type': 'string',
```

Comment:
I think either:
1) this should use the string_or_num from `schema/types.py`
or;
2) This should be `'type': 'number'` with a default value of 0.

It's a bit awkward to require it to be `'2'` instead of `2`.

Reply:
Thank you, fixed this to be string_or_num